static/app/views/issueDetails/streamline/flagSeries.tsx

```diff
@@ -110,6 +117,8 @@ export default function useFlagSeries({query = {}}: {query?: Record<string, any>
           '</div>',
           '<div class="tooltip-footer">',
           time,
+          event.dateCreated &&
```

Comment:
Does this && append the strings?

Reply:
it checks that `dateCreated` exists

Reply:
Oh so is the `time` on line 119 where we pass the absolute date? Could we get the date from the same source, `data.xAxis`?

Reply:
i think `time` is when the flag was created whereas `event.dateCreated` is when the event was created

Reply:
yup, @c298lee 's assumption is correct

Reply:
we want to show the flag's change/creation/deletion time in relation to the event creation date

Reply:
Oh I get it now! Thanks both of you